src/collective/behavior/amp/utils.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from collective.behavior.amp.config import AMP_VALID_ATTRIBUTES
+from lxml import html
+
+
+class Html2Amp:
+    def fix_img_tags(self, el):
```

Comment:
we can name this one as `replace_tags` or something like that as we're going to extend it in the near future.